### PR TITLE
Add Language history to agenda research

### PR DESCRIPTION
**Pull request by mjoyce91**
Added client's Language history to the Agenda Item research pane

Screen Shot 2022-02-03 at 12 37 10 PM

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
image
The second pipe looks a little different. If you want we can render the pipe on `useWrapper` instead of as a css border?

File: src/Components/ProfileDashboard/Languages/Languages.test.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    expect(wrapper.find('.languages-container').exists()).toBe(true);
+  });
+
+  it('does not render the .languages-container div when useWrapper is false', () => {
+    const wrapper = shallow(<Languages {...props} useWrapper={false} />);
+    expect(wrapper.find('.languages-container').exists()).toBe(false);
+  });
+
+  it('matches snapshot', () => {
+    const wrapper = shallow(<Languages content="content" {...props} />);
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Languages/Languages.test.jsx:40]: what does the `content="content"` do?

**Comment by elizabeth-jimenez** [comment]
Looks good, just a couple of comments 🦜

**Comment by mjoyce91** [comment]
> image
> 
> The second pipe looks a little different. If you want we can render the pipe on `useWrapper` instead of as a css border?

Yeah it's actually a border opposed to a pipe. The pipe seems more fitting on the left because it's separating similar data, but I chose the border for the date to kinda set it apart more. Maybe I could use parens for the date? Will play around with different options.

File: src/Components/ProfileDashboard/Languages/Languages.test.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/ProfileDashboard/Languages/Languages.test.jsx:40]: Looks like copy pasta

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - what about this?

Screen Shot 2022-02-08 at 9 13 11 AM

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez - what about this?
> 
> Screen Shot 2022-02-08 at 9 13 11 AM

Oh nice!

**Pull request merged**